conda_smithy/configure_feedstock.py

```diff
@@ -361,9 +361,6 @@ def render_appveyor(jinja_env, forge_config, forge_dir):
         for case in old_matrix:
             case = odict(case)
 
-            # Use Python 2.7 as a fallback when no Python version is set.
-            case["CONDA_PY"] = case.get("CONDA_PY", "27")
```

Comment:
This fallback was only needed to run the Obvious CI compiler activation script on AppVeyor. Examples of the sorts of cases where this fallback would occur include binary installs of prebuilt things on Windows or things built with MSYS2's compilers.

xref: https://github.com/pelson/Obvious-CI/blob/a0e09c1a199486e9d07c056bc40130a67c9e929f/scripts/obvci_appveyor_python_build_env.cmd#L1-L2